src/dbms/inmemory/replication_handlers.cpp

```diff
@@ -219,9 +220,12 @@ void InMemoryReplicationHandlers::SnapshotHandler(dbms::DbmsHandler *dbms_handle
     storage->timestamp_ = std::max(storage->timestamp_, recovery_info.next_timestamp);
 
     spdlog::trace("Recovering indices and constraints from snapshot.");
-    storage::durability::RecoverIndicesAndConstraints(recovered_snapshot.indices_constraints, &storage->indices_,
+    memgraph::storage::durability::RecoverIndicesAndStats(recovered_snapshot.indices_constraints.indices,
+                                                          &storage->indices_, &storage->vertices_,
+                                                          storage->name_id_mapper_.get(), std::nullopt);
+    memgraph::storage::durability::RecoverConstraints(recovered_snapshot.indices_constraints.constraints,
                                                       &storage->constraints_, &storage->vertices_,
-                                                      storage->name_id_mapper_.get());
+                                                      storage->name_id_mapper_.get(), std::nullopt);
```

Comment:
Can we default the last argument to nullopt, so we can skip defining it if don't want to do multi-threaded recovery.

Reply:
It is defaulted, but forgot to remove it